pkg/agent/openflow/client.go

```diff
@@ -825,12 +825,21 @@ func (c *client) InstallTraceflowFlows(dataplaneTag uint8) error {
 	for _, conj := range c.policyCache.List() {
 		for _, flow := range conj.(*policyRuleConjunction).metricFlows {
 			if flow.IsDropFlow() {
+				copyFlowBuilder := flow.CopyToBuilder(priorityNormal+2, false)
+				if flow.FlowProtocol() == "" {
+					copyFlowBuilder2 := flow.CopyToBuilder(priorityNormal+2, false)
```

Comment:
maybe `copyFlowBuilderIPv6` would be a better name here?

Reply:
Sure.